habitat/config/default.py

```diff
@@ -158,7 +161,6 @@
 _C.DATASET.POINTNAVV1.DATA_PATH = (
```

Comment:
Can we also do this change:
```suggestion
_C.DATASET.DATA_PATH = (
```